client/src/components/Header/index.jsx

```diff
@@ -17,7 +18,13 @@ export default function Header() {
 			</h1>
 			<nav aria-label="site navigation">
 				<ul>
-					<li>{principal && <NavLink to="/suggest">Suggest</NavLink>}</li>
+					<li>
+						{principal && (
+							<NavLink to="/suggest">
+								{user.is_admin ? "Publish" : "Suggest"}
+							</NavLink>
+						)}
+					</li>
```

Comment:
❓ This is interesting, not sure how I feel about this. You're right that the admin is no longer _suggesting_ as such, but the route is still `/suggest`. At this point maybe publish and suggest should be treated as separate routes and pages entirely, that just share the core form?

Reply:
ya I thought about that and about the component name 
But I did'nt want to make major updates on the code without reviewing the current